.github/scripts/attack.py

```
@@ -0,0 +1,341 @@
+    - Each time it connects, do a batch of UNSUB (free nodes) + SUB (allocate new nodes)
+    - Touch both SHARED_FILTERS and NOISE_FILTERS to create heavy free/alloc churn
+    """
+    log = mk_logger("CHURN")
+    active_noise: Set[str] = set(NOISE_FILTERS)  # Assume everything is present after seeding
+    active_shared: Set[str] = set(SHARED_FILTERS)
+
+    while not stop.is_set():
+        ev = threading.Event()
+        c = new_client(CLIENTID_X)
+
+        def on_connect(cl, userdata, flags, reason_code, properties):
```

Ruff (0.14.10)

Unused function argument: `cl`

(ARG001)